common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py

```diff
@@ -361,6 +412,32 @@ class TestSplitDirectOnlyCategorySemantics(DirectOnlyCategorySemantics):
     MODULESTORE = SPLIT_MODULESTORE_SETUP
     __test__ = True
 
+    @ddt.data(*TESTABLE_BLOCK_TYPES)
+    @XBlockAside.register_temp_plugin(AsideTest, 'test_aside')
+    @patch('xmodule.modulestore.split_mongo.caching_descriptor_system.CachingDescriptorSystem.applicable_aside_types',
+           lambda self, block: ['test_aside'])
+    def test_create_with_asides(self, block_type):
+        self._do_create(block_type, with_asides=True)
+
+    @ddt.data(*TESTABLE_BLOCK_TYPES)
+    @XBlockAside.register_temp_plugin(AsideTest, 'test_aside')
+    @patch('xmodule.modulestore.split_mongo.caching_descriptor_system.CachingDescriptorSystem.applicable_aside_types',
+           lambda self, block: ['test_aside'])
+    def test_update_asides(self, block_type):
+        block_usage_key = self._do_create(block_type, with_asides=True)
+        test_data = self.DATA_FIELDS[block_type]
+
+        with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred):
+            block = self.store.get_item(block_usage_key)
+            aside = self._get_aside(block)
+            self.assertIsNotNone(aside)
+            aside.fields[self.ASIDE_DATA_FIELD.field_name].write_to(aside, self.ASIDE_DATA_FIELD.updated)
+
+            self.store.update_item(block, ModuleStoreEnum.UserID.test, allow_not_found=True, asides=[aside])
+
+        self.assertBlockHasContent(block_usage_key, test_data.field_name, test_data.initial,
+                                   self.ASIDE_DATA_FIELD.field_name, self.ASIDE_DATA_FIELD.updated)
+
```

Comment:
Can you move these methods up to `DirectOnlyCategorySemantics`, so that we verify them for both split-mongo and old-mongo? (Or are asides only applicable to split?)